spec/lib/bamboozled/api/employee_spec.rb

```diff
@@ -123,16 +134,11 @@
       stub_request(:post, url).with(xml).to_return(response)
       employee = @client.employee.update("1234", details)
       expected_headers = {
-                           "content-type" => ["application/json; charset=utf-8"],
-                           "date" => ["Tue, 17 Jun 2014 19:25:35 UTC"]
-                         }
+        "content-type" => ["application/json; charset=utf-8"],
+        "date"         => ["Tue, 17 Jun 2014 19:25:35 UTC"]
```

Comment:
This is a default Hound rule that will be overwritten with our custom `rubocop` rules.